extensions/azdata/src/azdata.ts

```diff
@@ -49,10 +55,19 @@ class AzdataTool implements IAzdataTool {
 	}
 }
 
+export type AzdataLatestVersionInfo = {
```

Comment:
This is specific to brew right? Please indicate that in the name somehow - something like `AzdataDarwinPackageVersionInfo`